openedx/core/djangoapps/user_api/accounts/tests/test_utils.py

```diff
@@ -35,6 +35,7 @@ def validate_social_link(self, social_platform, link):
         ('twitter', 'https://www.twitter.com/edX/123s', None, False),
         ('twitter', 'twitter.com/edX', 'https://www.twitter.com/edX', True),
         ('twitter', 'twitter.com/edX?foo=bar', 'https://www.twitter.com/edX', True),
+        ('twitter', 'twitter.com/test.user', 'https://www.twitter.com/test.user', True),
```

Comment:
My issue was with Facebook. I verified that the only additional valid character is a period:

https://www.facebook.com/help/105399436216001

Reply:
LinkedIn doesn't seem to allow any special characters:

https://www.linkedin.com/help/linkedin/answer/87/customizing-your-public-profile-url?lang=en

Reply:
Same deal with Twitter: https://support.twitter.com/articles/101299